iocage_lib/ioc_start.py

```diff
@@ -728,7 +728,10 @@ def start_network(self, vnet):
             ]
 
         for nic in nics:
-            err = self.start_network_interface_vnet(nic, net_configs, jid)
+            if self.get("netgraph") == 'on':
```

Comment:
Bool PR got merged, this can now be
```suggestion
            if self.get("netgraph"):
```